src/Explorer/Controls/Settings/SettingsSubComponents/SubSettingsComponent.tsx

```diff
@@ -250,6 +250,23 @@ export class SubSettingsComponent extends React.Component<SubSettingsComponentPr
     </Stack>
   );
 
+  private getPartitionKeyValue = (): string => {
+    const partitionKeyValueProperties = this.props.collection.partitionKeyProperties;
+    if (!partitionKeyValueProperties || partitionKeyValueProperties.length === 0) {
+      return "";
+    }
+
+    let partitionKeyValue = "/" + partitionKeyValueProperties[0];
+
+    if (partitionKeyValueProperties.length > 1) {
+      for (let i = 1; i < partitionKeyValueProperties.length; i++) {
+        partitionKeyValue += ", /" + partitionKeyValueProperties[i];
+      }
+    }
```

Comment:
use 

  `partitionKeyValueProperties("/")`

this will work for empty arrays as well, so you can also remove the check for length==0 above

Reply:
Sorry, I meant,

`partitionKeyValueProperties.join('/')`

Reply:
Done